### ISSUE TITLE
MV repair stuck on 91% progress for more than 3 days

### ISSUE THREAD

**Issue by juliayakovlev** [scylladb/scylla-manager#3294]
## Issue description

- [ ] This issue is a regression.
- [x] It is unknown if this issue is a regression.

Repair task was created successfully:
```
< t:2023-01-27 06:54:08,224 f:base.py         l:142  c:RemoteLibSSH2CmdRunner p:DEBUG > Command "sudo sctool repair -c 8e101d3e-666c-4468-980e-7329a3229a72" finished with status 0
< t:2023-01-27 06:54:08,224 f:cli.py          l:615  c:sdcm.mgmt.cli        p:DEBUG > Created task id is: repair/5963781f-a6b6-42a9-ad5d-f36c93342f6d
```

The task is stuck on one materialized view repair: 
```
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > sctool output: Run:               6528f137-9e0f-11ed-be08-0a71663cabb9
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Status:           RUNNING
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Start time:       27 Jan 23 06:54:07 UTC
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Duration: 24h0m33s
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Progress: 99%
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Intensity:        1
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Parallel: 0
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Datacenters:      
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG >   - us-east
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > 
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ╭───────────────────────────────┬────────────────────────────────┬──────────┬───────────╮
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ Keyspace                      │                          Table │ Progress │ Duration  │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ keyspace1                     │                      standard1 │ 100%     │ 14m29s    │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ ks_truncate_large_partition   │                  test_counters │ 100%     │ 1m53s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ ks_truncate_large_partition   │                     test_table │ 100%     │ 4m48s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ mview                         │                          users │ 91%      │ 23h52m46s │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ mview                         │            users_by_first_name │ 100%     │ 8m56s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ mview                         │             users_by_last_name │ 91%      │ 23h54m21s │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_auth                   │                role_attributes │ 100%     │ 3m20s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_auth                   │                   role_members │ 100%     │ 2m43s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_auth                   │               role_permissions │ 100%     │ 3m4s      │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_auth                   │                          roles │ 100%     │ 2m54s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_distributed_everywhere │ cdc_generation_descriptions_v2 │ 100%     │ 2s        │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_distributed            │      cdc_generation_timestamps │ 100%     │ 2m28s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_distributed            │    cdc_streams_descriptions_v2 │ 100%     │ 4m26s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_distributed            │                 service_levels │ 100%     │ 4m21s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_distributed            │              view_build_status │ 100%     │ 4m19s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_traces                 │                         events │ 100%     │ 1m0s      │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_traces                 │                  node_slow_log │ 100%     │ 1m3s      │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_traces                 │         node_slow_log_time_idx │ 100%     │ 1m22s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_traces                 │                       sessions │ 100%     │ 1m14s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_traces                 │              sessions_time_idx │ 100%     │ 1m57s     │
< t:2023-01-28 06:54:41,332 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ╰───────────────────────────────┴────────────────────────────────┴──────────┴───────────╯
```

I checked the status at `30 Jan 23`, after 3 days from its start and it stack on the same progress:
```
ubuntu@longevity-tls-50gb-3d-master-monitor-node-91f4614d-1:~$ sudo sctool -c 8e101d3e-666c-4468-980e-7329a3229a72 task progress repair/5963781f-a6b6-42a9-ad5d-f36c93342f6d
Command "progress" is deprecated, use sctool backup|repair progress instead.

Run:		6528f137-9e0f-11ed-be08-0a71663cabb9
Status:		RUNNING
Start time:	27 Jan 23 06:54:07 UTC
Duration:	73h45m36s
Progress:	99%
Intensity:	1
Parallel:	0
Datacenters:	
  - us-east

╭───────────────────────────────┬────────────────────────────────┬──────────┬───────────╮
│ Keyspace                      │                          Table │ Progress │ Duration  │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ keyspace1                     │                      standard1 │ 100%     │ 14m29s    │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ ks_truncate_large_partition   │                  test_counters │ 100%     │ 1m53s     │
│ ks_truncate_large_partition   │                     test_table │ 100%     │ 4m48s     │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ mview                         │                          users │ 91%      │ 73h37m49s │
│ mview                         │            users_by_first_name │ 100%     │ 8m56s     │
│ mview                         │             users_by_last_name │ 91%      │ 73h39m25s │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ system_auth                   │                role_attributes │ 100%     │ 3m20s     │
│ system_auth                   │                   role_members │ 100%     │ 2m43s     │
│ system_auth                   │               role_permissions │ 100%     │ 3m4s      │
│ system_auth                   │                          roles │ 100%     │ 2m54s     │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ system_distributed_everywhere │ cdc_generation_descriptions_v2 │ 100%     │ 2s        │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ system_distributed            │      cdc_generation_timestamps │ 100%     │ 2m28s     │
│ system_distributed            │    cdc_streams_descriptions_v2 │ 100%     │ 4m26s     │
│ system_distributed            │                 service_levels │ 100%     │ 4m21s     │
│ system_distributed            │              view_build_status │ 100%     │ 4m19s     │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ system_traces                 │                         events │ 100%     │ 1m0s      │
│ system_traces                 │                  node_slow_log │ 100%     │ 1m3s      │
│ system_traces                 │         node_slow_log_time_idx │ 100%     │ 1m22s     │
│ system_traces                 │                       sessions │ 100%     │ 1m14s     │
│ system_traces                 │              sessions_time_idx │ 100%     │ 1m57s     │
╰───────────────────────────────┴────────────────────────────────┴──────────┴───────────╯
```

Nemesis:
```
< t:2023-01-27 06:53:43,662 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  > 2023-01-27 06:53:43.660: (DisruptionEvent Severity.NORMAL) period_type=begin event_id=c0a78f22-bf9c-43b1-b3c2-6bc9f7a7
9224: nemesis_name=MgmtRepairCli target_node=Node longevity-tls-50gb-3d-master-db-node-91f4614d-5 [54.226.109.147 | 10.12.10.123] (seed: False)
```

## How frequently does it reproduce?

Have no info about reproducing

## Installation details

Kernel Version: 5.15.0-1028-aws
Scylla version (or git commit hash): `5.3.0~dev-20230126.a0ff33e77774` with build-id `575b7a0891506801aeb1c05412a6eac011a0eda4`

Cluster size: 6 nodes (i4i.4xlarge)

Scylla Nodes used in this run:
- longevity-tls-50gb-3d-master-db-node-91f4614d-7 (3.95.18.168 | 10.12.9.13) (shards: -1)
- longevity-tls-50gb-3d-master-db-node-91f4614d-6 (52.55.169.59 | 10.12.8.241) (shards: 14)
- longevity-tls-50gb-3d-master-db-node-91f4614d-5 (54.226.109.147 | 10.12.10.123) (shards: 14)
- longevity-tls-50gb-3d-master-db-node-91f4614d-4 (54.196.151.77 | 10.12.9.157) (shards: 14)
- longevity-tls-50gb-3d-master-db-node-91f4614d-3 (54.234.230.24 | 10.12.11.159) (shards: 14)
- longevity-tls-50gb-3d-master-db-node-91f4614d-2 (54.167.109.174 | 10.12.8.179) (shards: 14)
- longevity-tls-50gb-3d-master-db-node-91f4614d-1 (54.158.3.176 | 10.12.10.158) (shards: 14)

OS / Image: `ami-0a8fd7e11139b6ca1` (aws: us-east-1)

Test: `longevity-50gb-3days`
Test id: `91f4614d-70d0-4ae2-9c9b-b9aa15d603bd`
Test name: `scylla-master/longevity/longevity-50gb-3days`
Test config file(s):

- [longevity-50GB-3days-authorization-and-tls-ssl.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/ce782d42d1aea97b0366fb51c80b70d521fdb749/test-cases/longevity/longevity-50GB-3days-authorization-and-tls-ssl.yaml)

<details>
<summary>
Logs and commands
</summary>

- Restore Monitor Stack command: `$ hydra investigate show-monitor 91f4614d-70d0-4ae2-9c9b-b9aa15d603bd`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=91f4614d-70d0-4ae2-9c9b-b9aa15d603bd)
- Show all stored logs command: `$ hydra investigate show-logs 91f4614d-70d0-4ae2-9c9b-b9aa15d603bd`

## Logs:
- **monitor-set-91f4614d.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230128_213155/monitor-set-91f4614d.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230128_213155/monitor-set-91f4614d.tar.gz)
- **loader-set-91f4614d.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230128_213155/loader-set-91f4614d.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230128_213155/loader-set-91f4614d.tar.gz)
- **monitor-set-91f4614d.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/monitor-set-91f4614d.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/monitor-set-91f4614d.tar.gz)
- **loader-set-91f4614d.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/loader-set-91f4614d.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/loader-set-91f4614d.tar.gz)

[Jenkins job URL](https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/longevity-50gb-3days/178/)
</details>

**Comment by roydahan** [comment]
@karol-kokoszka can you please have a look?
Maybe it's a dup of a known issue, but better to verify.

**Comment by karol-kokoszka** [comment]
@Michal-Leszczynski is already looking at it.

**Comment by juliayakovlev** [comment]
@Michal-Leszczynski 
Logs folder: 
https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/

**Comment by juliayakovlev** [comment]
@Michal-Leszczynski 
The cluster is alive. 
For DB nodes info see issue description.
Monitor:
longevity-tls-50gb-3d-master-monitor-node-91f4614d-1 |(54.88.119.6 | 10.12.8.93)

**Comment by Michal-Leszczynski** [comment]
This link does not seem to work for me:
https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/
But I found manager logs in monitor-set, so that's the interesting place to start from.
Anyways, thanks for the issue and the logs! I will try to find the source of this issue.

**Comment by Michal-Leszczynski** [comment]
Right now I'm focused on reading logs and repair implementation, so I guess that it can be terminated:)

**Comment by juliayakovlev** [comment]
Logs:
monitor-set: https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/monitor-set-91f4614d.tar.gz
db-cluster: https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/longevity-tls-50gb-3d-master-db-cluster-91f4614d.tar.gz
sct log: https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/sct.log.tar.gz

**Comment by ShlomiBalalis** [comment]
> Logs: monitor-set: https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/monitor-set-91f4614d.tar.gz db-cluster: https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/longevity-tls-50gb-3d-master-db-cluster-91f4614d.tar.gz sct log: https://cloudius-jenkins-test.s3.amazonaws.com/91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/sct.log.tar.gz

For some reason I don't understand you cannot download the db-cluster and sct logs through this url, you have to do it thtough the console: https://s3.console.aws.amazon.com/s3/buckets/cloudius-jenkins-test?region=us-east-1&prefix=91f4614d-70d0-4ae2-9c9b-b9aa15d603bd/20230129_164639/&showversions=false
You should have permission to do so
(There agent logs are combined with the db logs)

**Comment by Michal-Leszczynski** [comment]
I believe that this issue is caused by this fragment of the SM code:
https://github.com/scylladb/scylla-manager/blob/aa670a6aed10796ef92ce8708bae41e03238ce31/pkg/service/repair/worker.go#L211-L227
Here SM has a strange logic that it retires repair status query every time it exceeded it's timeout.
From logs of `longevity-tls-50gb-3d-master-db-node-91f4614d-4` we can see that agent receives status queries for jobs regarding `mview.users` and `mview.users_by_last_name` every 10 seconds (long polling timeout) till the end of log file.

I believe that load used in this test in combination with this [issue](https://github.com/scylladb/scylladb/issues/12693) overloads Scylla so much, that it is never able to handle repair status query within 10 seconds timeout and that's why SM is stuck in this loop.

The fix would be just to remove infinite retries after timeout is exceeded and end repair run with appropriate error message.

Should this fix be backported to previous versions of SM? @karol-kokoszka

**Comment by Michal-Leszczynski** [comment]
Looking at the logs from the second time this issue was observed, I can say that this is the same scenario - SM is stuck asking for repair status indefinitely while it does not get any response from Scylla node.

**Comment by Michal-Leszczynski** [comment]
@mykaul Should a fix for that be backported to previous versions of SM?
By fix I mean a change which makes SM end repair task with error instead of retrying to query repair status when long polling deadline is exceeded.

**Comment by mykaul** [comment]
> @mykaul Should a fix for that be backported to previous versions of SM? By fix I mean a change which makes SM end repair task with error instead of retrying to query repair status when long polling deadline is exceeded.

I don't think it is needed, if we get it into 3.1.

**Comment by roydahan** [comment]
> Looking at the logs from the second time this issue was observed, I can say that this is the same scenario - SM is stuck asking for repair status indefinitely while it does not get any response from Scylla node.

@Michal-Leszczynski it sounds like it needs a corresponding Scylla issue. Do we know why didn’t it get a response from the Scylla node?
(Leave the fact that Scylla manager should know how to handle it, we control and test both sides here).

**Comment by Michal-Leszczynski** [comment]
From the `messages.log from 16'th node` it looks like Scylla is to busy doing compaction, so it cannot respond to SM repair status query in time, but that is just my speculation, since I lack enough knowledge to debug it from Scylla core side.
Maybe it is expected behavior for such a big load?
Nevertheless, opening an issue in Scylla repo sounds like what should be done in this situation. There is an already open [issue](https://github.com/scylladb/scylladb/issues/12693) in there  (it's not really about repair, but it covers the same test scenario), so I will reference this issue there.

**Comment by Michal-Leszczynski** [comment]
Okay, I actually misinterpreted logs and how long polling works, sorry for that.
The code fragment linked in my previous answers is still does not make much sense and it can potentially lead to infinite loops when Scylla is not responding, but that's not the case here. 

In this case scenario SM keeps on asking Scylla about repair status every 10 seconds (long polling timeout), because Scylla is actually responding (in about exactly 10 seconds) with information that the job is still running, so long polling works as intended.
The problem is that Scylla seems to be stuck on this repair job and SM does not care about that and just waits for it to finish. So it looks like SM behavior is correct - SM repair task runs as long as Scylla repair task.

**Comment by Michal-Leszczynski** [comment]
I suggest to keep this issue open until we can get some confirmation on Scylla core side that I my speculations are correct.

**Comment by ilya-rarov** [comment]
Looks like we have it again on master (unfortunately, logs were not collected during this run)

## Issue description
Some details about running task
```
ubuntu@longevity-tls-50gb-3d-master-monitor-node-aa3c6baf-1:~$ sudo sctool tasks
Cluster: longevity-tls-50gb-3d-master-db-cluster-aa3c6baf (4136d110-f86f-426e-8575-cfe02343e8d5)
╭─────────────────────────────────────────────┬─────────────┬────────┬──────────┬─────────┬───────┬────────────────┬────────────┬─────────┬────────╮
│ Task                                        │ Schedule    │ Window │ Timezone │ Success │ Error │ Last Success   │ Last Error │ Status  │ Next   │
├─────────────────────────────────────────────┼─────────────┼────────┼──────────┼─────────┼───────┼────────────────┼────────────┼─────────┼────────┤
│ backup/4fd908f9-1e19-41b2-ad47-1c4aab49d0c0 │             │        │ Etc/UTC  │ 1       │ 0     │ 5d4h5m17s ago  │            │ DONE    │        │
│ healthcheck/cql                             │ @every 15s  │        │ Etc/UTC  │ 29802   │ 0     │ 7s ago         │            │ DONE    │ in 7s  │
│ healthcheck/rest                            │ @every 1m0s │        │ Etc/UTC  │ 7453    │ 0     │ 9s ago         │            │ DONE    │ in 50s │
│ healthcheck/alternator                      │ @every 15s  │        │ Etc/UTC  │ 29799   │ 0     │ 8s ago         │            │ DONE    │ in 6s  │
│ repair/04541c41-fa17-4081-af6a-9124d61edcf6 │             │        │ Etc/UTC  │ 0       │ 0     │                │            │ RUNNING │        │
│ repair/0874d925-e772-4b0f-9977-9d57dadc3205 │             │        │ Etc/UTC  │ 1       │ 0     │ 5d3h25m17s ago │            │ DONE    │        │
╰─────────────────────────────────────────────┴─────────────┴────────┴──────────┴─────────┴───────┴────────────────┴────────────┴─────────┴────────╯
```
```
ubuntu@longevity-tls-50gb-3d-master-monitor-node-aa3c6baf-1:~$ sudo sctool -c 4136d110-f86f-426e-8575-cfe02343e8d5 progress repair/04541c41-fa17-4081-af6a-9124d61edcf6
Run:		a504d62e-b416-11ed-94f4-0a52f18d1f2f
Status:		RUNNING
Start time:	24 Feb 23 07:41:26 UTC
Duration:	123h18m57s
Progress:	86%/3%
Intensity:	1
Parallel:	0
Datacenters:	
  - us-east

╭───────────────────────────────┬────────────────────────────────┬──────────┬────────────╮
│ Keyspace                      │                          Table │ Progress │ Duration   │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ keyspace1                     │                      standard1 │ 82%/5%   │ 123h17m45s │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ ks_truncate_large_partition   │                  test_counters │ 79%/5%   │ 2m57s      │
│ ks_truncate_large_partition   │                     test_table │ 79%/5%   │ 5m25s      │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ mview                         │                          users │ 69%/5%   │ 123h12m45s │
│ mview                         │            users_by_first_name │ 69%/5%   │ 7m11s      │
│ mview                         │             users_by_last_name │ 79%/5%   │ 7m28s      │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ system_auth                   │                role_attributes │ 94%      │ 3m53s      │
│ system_auth                   │                   role_members │ 94%      │ 3m44s      │
│ system_auth                   │               role_permissions │ 94%      │ 3m0s       │
│ system_auth                   │                          roles │ 94%      │ 3m22s      │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ system_distributed_everywhere │ cdc_generation_descriptions_v2 │ 0%       │ 0s         │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ system_distributed            │      cdc_generation_timestamps │ 89%/5%   │ 4m24s      │
│ system_distributed            │    cdc_streams_descriptions_v2 │ 89%/5%   │ 3m3s       │
│ system_distributed            │                 service_levels │ 89%/5%   │ 2m39s      │
│ system_distributed            │              view_build_status │ 94%      │ 4m40s      │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ system_traces                 │                         events │ 79%/6%   │ 98h58m2s   │
│ system_traces                 │                  node_slow_log │ 79%/7%   │ 98h57m40s  │
│ system_traces                 │         node_slow_log_time_idx │ 100%     │ 2m31s      │
│ system_traces                 │                       sessions │ 79%      │ 123h5m57s  │
│ system_traces                 │              sessions_time_idx │ 93%      │ 3m22s      │
╰───────────────────────────────┴────────────────────────────────┴──────────┴────────────╯
```
The error we got in SCT
```
Traceback (most recent call last):
  File "/home/ubuntu/scylla-cluster-tests/sdcm/wait.py", line 64, in wait_for
    res = retry(func, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/tenacity/__init__.py", line 404, in __call__
    do = self.iter(retry_state=retry_state)
  File "/usr/local/lib/python3.10/site-packages/tenacity/__init__.py", line 360, in iter
    raise retry_exc.reraise()
  File "/usr/local/lib/python3.10/site-packages/tenacity/__init__.py", line 194, in reraise
    raise self
tenacity.RetryError: RetryError[<Future at 0x7fd635438700 state=finished returned bool>]

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 4088, in wrapper
    result = method(*args[1:], **kwargs)
  File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/decorators.py", line 171, in wrapped
    res = func(*args, **kwargs)
  File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 2617, in disrupt_mgmt_repair_cli
    task_final_status = mgr_task.wait_and_get_final_status(timeout=86400)  # timeout is 24 hours
  File "/home/ubuntu/scylla-cluster-tests/sdcm/mgmt/cli.py", line 434, in wait_and_get_final_status
    res = self.wait_for_status(list_status=list_final_status, timeout=timeout, step=step)
  File "/home/ubuntu/scylla-cluster-tests/sdcm/mgmt/cli.py", line 398, in wait_for_status
    is_status_reached = wait.wait_for(func=self.is_status_in_list, step=step, throw_exc=True,
  File "/home/ubuntu/scylla-cluster-tests/sdcm/wait.py", line 75, in wait_for
    raise tenacity.RetryError(err) from ex
tenacity.RetryError: RetryError[Wait for: Waiting until task: repair/04541c41-fa17-4081-af6a-9124d61edcf6 reaches status of: ['ERROR', 'ERROR (4/4)', 'STOPPED', 'DONE', 'ABORTED']: timeout - 86400 seconds - expired]
```
## Installation details

Kernel Version: 5.15.0-1030-aws
Scylla version (or git commit hash): `5.3.0~dev-20230221.d7b6cf045fbf` with build-id `19fbad4a238b5ab9f8e00ca3aa25c940b4bce75d`




Cluster size: 6 nodes (i4i.4xlarge)

Scylla Nodes used in this run:
- longevity-tls-50gb-3d-master-db-node-aa3c6baf-7 (52.87.174.127 | 10.12.10.85) (shards: -1)
- longevity-tls-50gb-3d-master-db-node-aa3c6baf-6 (3.93.31.229 | 10.12.8.138) (shards: 14)
- longevity-tls-50gb-3d-master-db-node-aa3c6baf-5 (54.167.20.129 | 10.12.8.181) (shards: 14)
- longevity-tls-50gb-3d-master-db-node-aa3c6baf-4 (18.206.199.93 | 10.12.10.207) (shards: 14)
- longevity-tls-50gb-3d-master-db-node-aa3c6baf-3 (54.144.101.131 | 10.12.10.13) (shards: 14)
- longevity-tls-50gb-3d-master-db-node-aa3c6baf-2 (54.160.63.59 | 10.12.11.157) (shards: 14)
- longevity-tls-50gb-3d-master-db-node-aa3c6baf-1 (34.234.77.209 | 10.12.9.163) (shards: 14)


OS / Image: `ami-065d0cb40cc02d82c` (aws: us-east-1)

Test: `longevity-50gb-3days-test`
Test id: `aa3c6baf-5ee1-4aab-b552-e3975635ecb6`
Test name: `scylla-master/longevity/longevity-50gb-3days-test`
Test config file(s):

- [longevity-50GB-3days-authorization-and-tls-ssl.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/7f6ed83301320a1d9bf73ee4420f07e9422f3c3e/test-cases/longevity/longevity-50GB-3days-authorization-and-tls-ssl.yaml)


<details>
<summary>
Logs and commands
</summary>


- Restore Monitor Stack command: `$ hydra investigate show-monitor aa3c6baf-5ee1-4aab-b552-e3975635ecb6`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=aa3c6baf-5ee1-4aab-b552-e3975635ecb6)
- Show all stored logs command: `$ hydra investigate show-logs aa3c6baf-5ee1-4aab-b552-e3975635ecb6`


## Logs:
*No logs captured during this run.*

[Jenkins job URL](https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/longevity-50gb-3days-test/183/)
</details>

**Comment by roydahan** [comment]
> I suggest to keep this issue open until we can get some confirmation on Scylla core side that I my speculations are correct.

@Michal-Leszczynski what is the issue number on scylla core side?

**Comment by Michal-Leszczynski** [comment]
Well, I pasted my observations as a [comment](https://github.com/scylladb/scylladb/issues/12693#issuecomment-1433244239) to the scylla core issue mentioned above. I asked if they think that it requires separate issue, but I didn't get any response yet.

**Comment by yarongilor** [comment]
Reproduced in another test (ics 1tb 5d):


## Issue description

- [ ] This issue is a regression.
- [ ] It is unknown if this issue is a regression.

*Describe your issue in detail and steps it took to produce it.*

## Impact

log:
```
< t:2023-03-07 21:25:13,029 f:base.py         l:142  c:RemoteLibSSH2CmdRunner p:DEBUG > Command "sudo sctool  -c 9281ad29-ba30-4ba8-890c-62dcf49331bd progress repair/fec5224b-a534-4cbb-bd4a-3b63dd85c239" finished with status 0
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > sctool output: Run:		0fe3c571-bd0c-11ed-8fb6-0a86958a490f
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Status:		RUNNING
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Start time:	07 Mar 23 17:18:22 UTC
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Duration:	4h6m50s
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Progress:	95%
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Intensity:	1
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Parallel:	0
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > Datacenters:	
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG >   - eu-west
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > 
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ╭───────────────────────────────┬────────────────────────────────┬──────────┬──────────╮
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ Keyspace                      │                          Table │ Progress │ Duration │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼──────────┤
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ keyspace1                     │                      standard1 │ 100%     │ 3h18m17s │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼──────────┤
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ keyspace_deflate              │                      standard1 │ 100%     │ 22m14s   │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼──────────┤
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ keyspace_lz4                  │                      standard1 │ 100%     │ 8m4s     │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼──────────┤
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ keyspace_snappy               │                      standard1 │ 89%      │ 9m38s    │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼──────────┤
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ mview                         │                          users │ 74%      │ 9m39s    │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ mview                         │            users_by_first_name │ 74%      │ 5m7s     │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ mview                         │             users_by_last_name │ 74%      │ 12m41s   │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼──────────┤
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_auth                   │                role_attributes │ 100%     │ 8m10s    │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_auth                   │                   role_members │ 100%     │ 8m39s    │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_auth                   │               role_permissions │ 100%     │ 8m5s     │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_auth                   │                          roles │ 100%     │ 8m3s     │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼──────────┤
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_distributed_everywhere │ cdc_generation_descriptions_v2 │ 100%     │ 0s       │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼──────────┤
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_distributed            │      cdc_generation_timestamps │ 100%     │ 10m34s   │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_distributed            │    cdc_streams_descriptions_v2 │ 100%     │ 10m22s   │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_distributed            │                 service_levels │ 100%     │ 9m36s    │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_distributed            │              view_build_status │ 100%     │ 10m4s    │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ├───────────────────────────────┼────────────────────────────────┼──────────┼──────────┤
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_traces                 │                         events │ 100%     │ 2m57s    │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_traces                 │                  node_slow_log │ 100%     │ 2m26s    │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_traces                 │         node_slow_log_time_idx │ 100%     │ 2m55s    │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_traces                 │                       sessions │ 100%     │ 2m4s     │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > │ system_traces                 │              sessions_time_idx │ 100%     │ 2m42s    │
< t:2023-03-07 21:25:13,029 f:cli.py          l:1088 c:sdcm.mgmt.cli        p:DEBUG > ╰───────────────────────────────┴────────────────────────────────┴──────────┴──────────╯
```


## How frequently does it reproduce?

*Describe the frequency with how this issue can be reproduced.*

## Installation details

Kernel Version: 5.15.0-1031-aws
Scylla version (or git commit hash): `2023.1.0~rc2-20230302.726f8a090337` with build-id `efcc950cda22875aa488c5295c6cc68f35d9cc6f`




Cluster size: 4 nodes (i3.4xlarge)

Scylla Nodes used in this run:
- longevity-tls-1tb-7d-2023-1-db-node-52776f27-8 (54.247.18.109 | 10.4.1.79) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-52776f27-7 (3.253.57.198 | 10.4.3.240) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-52776f27-6 (34.247.183.17 | 10.4.0.35) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-52776f27-5 (3.250.177.43 | 10.4.2.134) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-52776f27-4 (54.75.80.9 | 10.4.1.130) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-52776f27-3 (54.72.86.47 | 10.4.1.247) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-52776f27-2 (34.244.176.55 | 10.4.3.44) (shards: 14)
- longevity-tls-1tb-7d-2023-1-db-node-52776f27-1 (34.245.12.22 | 10.4.0.24) (shards: 14)


OS / Image: `ami-0336a5a646b08dc25` (aws: eu-west-1)

Test: `ics-longevity-1tb-7days-test`
Test id: `52776f27-b99b-444c-982e-89fbb62999ad`
Test name: `enterprise-2023.1/SCT_Enterprise_Features/ICS/ics-longevity-1tb-7days-test`
Test config file(s):

- [longevity-1TB-5days-authorization-and-tls-ssl.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/9a5f163455e77d1518d00ea2a5ac14df134cf2f6/test-cases/longevity/longevity-1TB-5days-authorization-and-tls-ssl.yaml)


<details>
<summary>
Logs and commands
</summary>


- Restore Monitor Stack command: `$ hydra investigate show-monitor 52776f27-b99b-444c-982e-89fbb62999ad`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=52776f27-b99b-444c-982e-89fbb62999ad)
- Show all stored logs command: `$ hydra investigate show-logs 52776f27-b99b-444c-982e-89fbb62999ad`


## Logs:
- **db-cluster-52776f27.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/db-cluster-52776f27.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/db-cluster-52776f27.tar.gz)
- **output-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/output-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/output-52776f27.log.tar.gz)
- **debug-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/debug-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/debug-52776f27.log.tar.gz)
- **events-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/events-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/events-52776f27.log.tar.gz)
- **sct-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/sct-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/sct-52776f27.log.tar.gz)
- **normal-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/normal-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/normal-52776f27.log.tar.gz)
- **argus-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/argus-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/argus-52776f27.log.tar.gz)
- **raw_events-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/raw_events-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/raw_events-52776f27.log.tar.gz)
- **critical-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/critical-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/critical-52776f27.log.tar.gz)
- **warning-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/warning-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/warning-52776f27.log.tar.gz)
- **summary-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/summary-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/summary-52776f27.log.tar.gz)
- **error-52776f27.log.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/error-52776f27.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/error-52776f27.log.tar.gz)
- **monitor-set-52776f27.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/monitor-set-52776f27.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/monitor-set-52776f27.tar.gz)
- **loader-set-52776f27.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/loader-set-52776f27.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/52776f27-b99b-444c-982e-89fbb62999ad/20230307_211805/loader-set-52776f27.tar.gz)


[Jenkins job URL](https://jenkins.scylladb.com/job/enterprise-2023.1/job/SCT_Enterprise_Features/job/ICS/job/ics-longevity-1tb-7days-test/2/)
</details>

**Comment by ilya-rarov** [comment]
Got it again (logs were not collected).


## Issue description
Stuck task details
```
ubuntu@longevity-tls-50gb-3d-2023-1-monitor-node-e441a547-1:~$ sudo sctool tasks
Cluster: longevity-tls-50gb-3d-2023-1-db-cluster-e441a547 (ea5aa3ae-dcd5-45a6-8827-15ed83ec47bc)
╭─────────────────────────────────────────────┬─────────────┬────────┬──────────┬─────────┬───────┬──────────────┬────────────┬─────────┬────────╮
│ Task                                        │ Schedule    │ Window │ Timezone │ Success │ Error │ Last Success │ Last Error │ Status  │ Next   │
├─────────────────────────────────────────────┼─────────────┼────────┼──────────┼─────────┼───────┼──────────────┼────────────┼─────────┼────────┤
│ healthcheck/cql                             │ @every 15s  │        │ Etc/UTC  │ 11147   │ 0     │ 6s ago       │            │ DONE    │ in 8s  │
│ healthcheck/alternator                      │ @every 15s  │        │ Etc/UTC  │ 11150   │ 0     │ 6s ago       │            │ DONE    │ in 8s  │
│ healthcheck/rest                            │ @every 1m0s │        │ Etc/UTC  │ 2788    │ 0     │ 27s ago      │            │ DONE    │ in 32s │
│ repair/3b1a3c7d-c448-4b62-a7db-63d67c8520c5 │             │        │ Etc/UTC  │ 0       │ 0     │              │            │ RUNNING │        │
╰─────────────────────────────────────────────┴─────────────┴────────┴──────────┴─────────┴───────┴──────────────┴────────────┴─────────┴────────╯
```
```
ubuntu@longevity-tls-50gb-3d-2023-1-monitor-node-e441a547-1:~$ sudo sctool -c ea5aa3ae-dcd5-45a6-8827-15ed83ec47bc progress repair/3b1a3c7d-c448-4b62-a7db-63d67c8520c5
Run:		b0aca28b-c28c-11ed-8e0a-0a54a74aa2d9
Status:		RUNNING
Start time:	14 Mar 23 17:21:43 UTC
Duration:	46h29m53s
Progress:	66%/0%
Intensity:	1
Parallel:	0
Datacenters:	
  - eu-west

╭───────────────────────────────┬────────────────────────────────┬──────────┬───────────╮
│ Keyspace                      │                          Table │ Progress │ Duration  │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ keyspace1                     │                      standard1 │ 62%      │ 46h29m4s  │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ mview                         │                          users │ 39%      │ 46h25m26s │
│ mview                         │            users_by_first_name │ 49%      │ 46h24m3s  │
│ mview                         │             users_by_last_name │ 28%/4%   │ 46h26m39s │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ system_auth                   │                role_attributes │ 77%      │ 3m51s     │
│ system_auth                   │                   role_members │ 72%      │ 3m22s     │
│ system_auth                   │               role_permissions │ 82%      │ 4m8s      │
│ system_auth                   │                          roles │ 82%      │ 3m33s     │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ system_distributed_everywhere │ cdc_generation_descriptions_v2 │ 0%       │ 0s        │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ system_distributed            │      cdc_generation_timestamps │ 72%      │ 2m21s     │
│ system_distributed            │    cdc_streams_descriptions_v2 │ 72%      │ 2m28s     │
│ system_distributed            │                 service_levels │ 72%      │ 2m41s     │
│ system_distributed            │              view_build_status │ 72%      │ 2m15s     │
├───────────────────────────────┼────────────────────────────────┼──────────┼───────────┤
│ system_traces                 │                         events │ 67%      │ 5m26s     │
│ system_traces                 │                  node_slow_log │ 67%      │ 42s       │
│ system_traces                 │         node_slow_log_time_idx │ 67%      │ 46h26m36s │
│ system_traces                 │                       sessions │ 79%      │ 46h25m45s │
│ system_traces                 │              sessions_time_idx │ 67%      │ 46h25m53s │
╰───────────────────────────────┴────────────────────────────────┴──────────┴───────────╯
```
The errorr in SCT
```
Traceback (most recent call last):
  File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 4198, in wrapper
    result = method(*args[1:], **kwargs)
  File "/home/ubuntu/scylla-cluster-tests/sdcm/utils/decorators.py", line 171, in wrapped
    res = func(*args, **kwargs)
  File "/home/ubuntu/scylla-cluster-tests/sdcm/nemesis.py", line 2643, in disrupt_mgmt_repair_cli
    task_final_status = mgr_task.wait_and_get_final_status(timeout=86400)  # timeout is 24 hours
  File "/home/ubuntu/scylla-cluster-tests/sdcm/mgmt/cli.py", line 434, in wait_and_get_final_status
    res = self.wait_for_status(list_status=list_final_status, timeout=timeout, step=step)
  File "/home/ubuntu/scylla-cluster-tests/sdcm/mgmt/cli.py", line 398, in wait_for_status
    is_status_reached = wait.wait_for(func=self.is_status_in_list, step=step, throw_exc=True,
  File "/home/ubuntu/scylla-cluster-tests/sdcm/wait.py", line 75, in wait_for
    raise tenacity.RetryError(err) from ex
tenacity.RetryError: RetryError[Wait for: Waiting until task: repair/3b1a3c7d-c448-4b62-a7db-63d67c8520c5 reaches status of: ['ERROR', 'ERROR (4/4)', 'STOPPED', 'DONE', 'ABORTED']: timeout - 86400 seconds - expired]
```

## Installation details

Kernel Version: 5.15.0-1031-aws
Scylla version (or git commit hash): `2023.1.0~rc2-20230302.726f8a090337` with build-id `efcc950cda22875aa488c5295c6cc68f35d9cc6f`




Cluster size: 6 nodes (i4i.4xlarge)

Scylla Nodes used in this run:
- longevity-tls-50gb-3d-2023-1-db-node-e441a547-8 (3.251.96.227 | 10.4.3.207) (shards: -1)
- longevity-tls-50gb-3d-2023-1-db-node-e441a547-7 (34.245.115.223 | 10.4.0.118) (shards: 14)
- longevity-tls-50gb-3d-2023-1-db-node-e441a547-6 (52.16.195.148 | 10.4.2.245) (shards: 14)
- longevity-tls-50gb-3d-2023-1-db-node-e441a547-5 (34.243.172.50 | 10.4.0.176) (shards: 14)
- longevity-tls-50gb-3d-2023-1-db-node-e441a547-4 (54.154.111.26 | 10.4.2.150) (shards: 14)
- longevity-tls-50gb-3d-2023-1-db-node-e441a547-3 (54.154.144.92 | 10.4.0.206) (shards: 14)
- longevity-tls-50gb-3d-2023-1-db-node-e441a547-2 (34.254.187.129 | 10.4.3.51) (shards: 14)
- longevity-tls-50gb-3d-2023-1-db-node-e441a547-1 (34.245.131.144 | 10.4.1.10) (shards: 14)


OS / Image: `ami-0336a5a646b08dc25` (aws: eu-west-1)

Test: `longevity-50gb-3days-aws-i4i-test`
Test id: `e441a547-b41f-4c6f-a4a9-8f1fea4a9826`
Test name: `enterprise-2023.1/longevity/longevity-50gb-3days-aws-i4i-test`
Test config file(s):

- [longevity-50GB-3days-authorization-and-tls-ssl.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/4ec1600f015410814cdbe6d94ce29af866d6ccda/test-cases/longevity/longevity-50GB-3days-authorization-and-tls-ssl.yaml)


<details>
<summary>
Logs and commands
</summary>


- Restore Monitor Stack command: `$ hydra investigate show-monitor e441a547-b41f-4c6f-a4a9-8f1fea4a9826`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=e441a547-b41f-4c6f-a4a9-8f1fea4a9826)
- Show all stored logs command: `$ hydra investigate show-logs e441a547-b41f-4c6f-a4a9-8f1fea4a9826`


## Logs:
*No logs captured during this run.*

[Jenkins job URL](https://jenkins.scylladb.com/job/enterprise-2023.1/job/longevity/job/longevity-50gb-3days-aws-i4i-test/3/)
</details>

**Comment by roydahan** [comment]
@juliayakovlev does your reproducer suggests that this issue still happens on latest master??

**Comment by juliayakovlev** [comment]
> @juliayakovlev does your reproducer suggests that this issue still happens on latest master??

@roydahan @Michal-Leszczynski 

Now test is running with Scylla version: `5.3.0~dev-20230407.c65bd01174c1`
Nemesis `disrupt_mgmt_repair_cli` was started at `2023-04-10 15:24:34` and failed at `2023-04-11 15:25:09` (**4.5 days** ago) with error:
```
tenacity.RetryError: RetryError[Wait for: Waiting until task: repair/c27f17c9-2ccd-4920-9d38-183bb330e5f5 reaches status of: ['ERROR', 'ERROR (4/4)', 'STOPPED', 'DONE', 'ABORTED']: timeout - 86400 seconds - expired]
```

The task status now, `2023-04-16`:
```
ubuntu@longevity-tls-50gb-3d-reproduc-monitor-node-0ea9ece0-1:~$ sudo sctool -c f8888003-68d6-4fd0-b12e-5b1fc0bcead4 task progress repair/c27f17c9-2ccd-4920-9d38-183bb330e5f5
Command "progress" is deprecated, use sctool backup|repair progress instead.

Run:		d9505ae9-d7b3-11ed-8a66-0268d9727f71
Status:		RUNNING
Start time:	10 Apr 23 15:24:56 UTC
Duration:	135h51m56s
Progress:	92%/4%
Intensity:	1
Parallel:	0
Datacenters:	
  - UNKNOWN_DC
  - us-east

╭───────────────────────────────┬────────────────────────────────┬──────────┬────────────╮
│ Keyspace                      │                          Table │ Progress │ Duration   │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ keyspace1                     │                      standard1 │ 96%/4%   │ 111h46m5s  │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ mview                         │                          users │ 80%/4%   │ 135h50m54s │
│ mview                         │            users_by_first_name │ 80%/4%   │ 135h51m4s  │
│ mview                         │             users_by_last_name │ 80%/4%   │ 135h51m11s │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ system_auth                   │                role_attributes │ 100%     │ 14s        │
│ system_auth                   │                   role_members │ 96%/4%   │ 24h5m44s   │
│ system_auth                   │               role_permissions │ 96%/4%   │ 24h5m47s   │
│ system_auth                   │                          roles │ 96%/4%   │ 24h5m48s   │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ system_distributed_everywhere │ cdc_generation_descriptions_v2 │ 0%       │ 111h44m57s │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ system_distributed            │      cdc_generation_timestamps │ 96%/4%   │ 24h5m46s   │
│ system_distributed            │    cdc_streams_descriptions_v2 │ 96%/4%   │ 24h5m46s   │
│ system_distributed            │                 service_levels │ 96%/4%   │ 24h5m51s   │
│ system_distributed            │              view_build_status │ 96%/4%   │ 24h5m47s   │
├───────────────────────────────┼────────────────────────────────┼──────────┼────────────┤
│ system_traces                 │                         events │ 80%/8%   │ 24h5m31s   │
│ system_traces                 │                  node_slow_log │ 91%/8%   │ 24h5m36s   │
│ system_traces                 │         node_slow_log_time_idx │ 100%     │ 8s         │
│ system_traces                 │                       sessions │ 91%/8%   │ 24h5m37s   │
│ system_traces                 │              sessions_time_idx │ 100%     │ 8s         │
╰───────────────────────────────┴────────────────────────────────┴──────────┴────────────╯
```

Cluster is alive and monistor is up as well:
`longevity-tls-50gb-3d-reproduc-monitor-node-0ea9ece0-1 (3.235.98.71 | 10.12.2.88`)`




## Issue description

- [ ] This issue is a regression.
- [ ] It is unknown if this issue is a regression.

*Describe your issue in detail and steps it took to produce it.*

## Impact

*Describe the impact this issue causes to the user.*

## How frequently does it reproduce?

*Describe the frequency with how this issue can be reproduced.*

## Installation details

Kernel Version: 5.19.0-1022-aws
Scylla version (or git commit hash): `5.3.0~dev-20230407.c65bd01174c1` with build-id `30cf2f853017f7b98744f6ca55e1df0779aa6835`




Cluster size: 6 nodes (i4i.4xlarge)

Scylla Nodes used in this run:
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-9 (44.215.125.27 | 10.12.1.117) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-8 (3.236.129.235 | 10.12.0.252) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-7 (18.215.189.128 | 10.12.0.25) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-6 (3.235.158.51 | 10.12.3.114) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-5 (34.200.229.53 | 10.12.0.202) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-4 (44.200.214.7 | 10.12.1.58) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-3 (3.238.183.194 | 10.12.1.186) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-2 (44.215.70.25 | 10.12.3.133) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-15 (44.199.233.95 | 10.12.3.68) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-14 (44.214.15.195 | 10.12.1.121) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-13 (3.236.153.119 | 10.12.0.181) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12 (44.199.244.24 | 10.12.0.246) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-11 (3.230.163.185 | 10.12.1.192) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-10 (100.24.113.195 | 10.12.3.242) (shards: 14)
- longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-1 (3.238.154.39 | 10.12.2.158) (shards: 14)


OS / Image: `ami-0c6d8fcb02644262c` (aws: us-east-1)

Test: `longevity-50gb-6days-i12693`
Test id: `0ea9ece0-e184-4f9f-a4a0-c4a27a08d5e4`
Test name: `scylla-master/reproducers/longevity-50gb-6days-i12693`
Test config file(s):

- [longevity-50GB-3days-authorization-and-tls-ssl.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/6c2e9f7f6fe32c8dd6297391c2398a171628abbd/test-cases/longevity/longevity-50GB-3days-authorization-and-tls-ssl.yaml)


<details>
<summary>
Logs and commands
</summary>


- Restore Monitor Stack command: `$ hydra investigate show-monitor 0ea9ece0-e184-4f9f-a4a0-c4a27a08d5e4`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=0ea9ece0-e184-4f9f-a4a0-c4a27a08d5e4)
- Show all stored logs command: `$ hydra investigate show-logs 0ea9ece0-e184-4f9f-a4a0-c4a27a08d5e4`


## Logs:
*No logs captured during this run.*

[Jenkins job URL](https://jenkins.scylladb.com/job/scylla-master/job/reproducers/job/longevity-50gb-6days-i12693/5/)
</details>

**Comment by juliayakovlev** [comment]
> @juliayakovlev is the cluster still alive? 
No, cluster was terminated. All logs are saved

>> Does someone look/ed at it?

I am not aware that someone looks on it

> 
> @Michal-Leszczynski ?

**Comment by tzach** [comment]
@mykaul this is for *repair* not *restore*

**Comment by mykaul** [comment]
> @mykaul this is for _repair_ not _restore_

We repair what we've restored, no? Which I assume we should not be doing, but recreate it?

**Comment by Michal-Leszczynski** [comment]
Sorry that this issue hasn't gotten any attention since the rerun of the tests!

From the logs we see that the repair runs fine up 92% progress.
At this point, we can see some problems with host 10.12.0.246 - workers that await the end of repairs which master is 10.12.0.246 are stuck for about 24h while constantly getting responses from Scylla that the repairs are still running.
```
2023-04-10T15:26:27+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-10T15:26:27.343Z","N":"http","M":"GET /storage_service/repair_status?id=1196&timeout=10","from":"10.12.2.88:45334","status":200,"bytes":9,"duration":"10000ms"}
2023-04-10T15:26:37+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-10T15:26:37.343Z","N":"http","M":"GET /storage_service/repair_status?id=1196&timeout=10","from":"10.12.2.88:45334","status":200,"bytes":9,"duration":"10000ms"}
...
2023-04-11T15:31:26+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-11T15:31:26.075Z","N":"http","M":"GET /storage_service/repair_status?id=1196&timeout=10","from":"10.12.2.88:45322","status":200,"bytes":9,"duration":"10000ms"}
2023-04-11T15:31:36+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-11T15:31:36.077Z","N":"http","M":"GET /storage_service/repair_status?id=1196&timeout=10","from":"10.12.2.88:42240","status":200,"bytes":9,"duration":"10000ms"}
2023-04-11T15:31:46+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-11T15:31:46.078Z","N":"http","M":"GET /storage_service/repair_status?id=1196&timeout=10","from":"10.12.2.88:48094","status":200,"bytes":9,"duration":"10000ms"}
```

After 24h, we can see that node 10.12.0.246 encounters many errors, which informs repair workers about the failure:
```
2023-04-11T15:31:45+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12  !WARNING | scylla[5387]:  [shard  4] storage_proxy - Failed to apply mutation from 10.12.2.158#4: seastar::abort_requested_exception (abort requested)
2023-04-11T15:31:45+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12  !WARNING | scylla[5387]:  [shard  0] storage_proxy - Failed to apply mutation from 10.12.3.133#0: seastar::abort_requested_exception (abort requested)
2023-04-11T15:31:45+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12  !WARNING | scylla[5387]:  [shard  7] storage_proxy - Failed to apply mutation from 10.12.3.68#7: seastar::abort_requested_exception (abort requested)
```
The repair wasn't scheduled with the `--fail-fast` flag, so it still tries to repair other token ranges.

```
Apr 11 15:32:01  {"L":"INFO","T":"2023-04-11T15:32:01.138Z","N":"repair.worker 26","M":"Repairing","keyspace":"keyspace1","table":"standard1","hosts":["10.12.0.246","10.12.3.133","10.12.2.158"],"ranges":1,"master":"10.12.0.246","job_id":1,"_trace_id":"ZTfTVK0HTeuIVUoFLr5zYg"}
Apr 11 15:32:01  {"L":"INFO","T":"2023-04-11T15:32:01.548Z","N":"repair.worker 20","M":"Repairing","keyspace":"keyspace1","table":"standard1","hosts":["10.12.0.246","10.12.3.133","10.12.2.158"],"ranges":1,"master":"10.12.0.246","job_id":2,"_trace_id":"ZTfTVK0HTeuIVUoFLr5zYg"}
Apr 11 15:32:02  {"L":"INFO","T":"2023-04-11T15:32:02.689Z","N":"repair.worker 36","M":"Repairing","keyspace":"keyspace1","table":"standard1","hosts":["10.12.3.133","10.12.0.246","10.12.2.158"],"ranges":1,"master":"10.12.0.246","job_id":3,"_trace_id":"ZTfTVK0HTeuIVUoFLr5zYg"}
```
We can see that `job_id` was reset (it starts from 1) on node 10.12.0.246 (probably due to node restart or something similar). Now repair workers wait for those new repairs, but node 10.12.0.246 keeps on responding that those repairs are still running even after a few days:

```
2023-04-11T15:32:13+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-11T15:32:12.764Z","N":"http","M":"GET /storage_service/repair_status?id=4&timeout=10","from":"10.12.2.88:49820","status":200,"bytes":9,"duration":"10000ms"}
2023-04-11T15:32:23+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-11T15:32:22.765Z","N":"http","M":"GET /storage_service/repair_status?id=4&timeout=10","from":"10.12.2.88:49820","status":200,"bytes":9,"duration":"10000ms"}
2023-04-11T15:32:33+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-11T15:32:32.766Z","N":"http","M":"GET /storage_service/repair_status?id=4&timeout=10","from":"10.12.2.88:49820","status":200,"bytes":9,"duration":"10000ms"}
...
2023-04-16T12:20:15+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-16T12:20:15.482Z","N":"http","M":"GET /storage_service/repair_status?id=4&timeout=10","from":"10.12.2.88:48076","status":200,"bytes":9,"duration":"10000ms"}
2023-04-16T12:20:25+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-16T12:20:25.483Z","N":"http","M":"GET /storage_service/repair_status?id=4&timeout=10","from":"10.12.2.88:48076","status":200,"bytes":9,"duration":"10000ms"}
2023-04-16T12:20:35+00:00 longevity-tls-50gb-3d-reproduc-db-node-0ea9ece0-12     !INFO | scylla-manager-agent[6601]: {"L":"INFO","T":"2023-04-16T12:20:35.484Z","N":"http","M":"GET /storage_service/repair_status?id=4&timeout=10","from":"10.12.2.88:48076","status":200,"bytes":9,"duration":"10000ms"}
```
And that's the end of logs. So as far as I understand SM POV, this is expected behavior?

Having said that, right now we are working on refactoring repair procedure so that it would repair table by table (not all tables in parallel) in order to boost its performance. If the root cause of this issue is that SM overloads node 10.12.0.246 with too many repairs, this might be the solution.
These are just my speculations, so it would be great to get some interpretation (from core side) of why does node 10.12.0.246 gets stuck at repairing some token ranges for so many days.

Again, sorry for such a late response.

**Comment by mykaul** [comment]
Since QA sees it happening from time to time, let's reproduce in 3.1.1 when we have the table by table repair included.

**Comment by mykaul** [comment]
@juliayakovlev - can you please open a clean bug on this issue on scylla core? This needs to be investigated not in manager (I don't want to move over as there's plenty of discussion here).

**Comment by mykaul** [comment]
> @juliayakovlev - can you please open a clean bug on this issue on scylla core? This needs to be investigated not in manager (I don't want to move over as there's plenty of discussion here).

@roydahan - was there a new bug opened? Shall we transfer this one (too much clutter...) ?

**Comment by roydahan** [comment]
I don't see new one, but I think most of the information here is helpful and having the history in place will be helpful.
